oam/install_scripts/post-mysql-install

```diff
@@ -103,11 +105,11 @@ if [ -x $installdir/mysql/mysql-Columnstore ]; then
 	sleep 5
 
 	# Install various Calpont stuff...
-	$installdir/mysql/install_calpont_mysql.sh  --installdir=$installdir
+	$installdir/mysql/install_calpont_mysql.sh  --installdir=$installdir --tmpdir=$tmpdir
 	checkForError
```

Comment:
Unrelated to the patch.  I doubt a password-related error is the only thing that can go wrong here.